### PR TITLE
Made staging deploy easier to distinguish from demo.

### PR DESCRIPTION
**Pull request by benwarfield-usds**
Used environment-passing tricks to make the staging environment have a different window title from the demo environment.

If I'm feeling lucky, we may also get a different app icon for it.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -34,6 +45,9 @@ jobs:
     - name: Application build
       working-directory: ./frontend
       run: npm run-script build
+      env:
+        REACT_APP_TITLE: $${{env.APP_TITLE}}
+        REACT_APP_ICON: optimus_prime.svg
```
Comment (benwarfield-usds) [inline .github/workflows/deploy.yml:50]: Well, that's stupid, Ben.

**Review by benwarfield-usds — COMMENTED** [review]
Gotta fix the icon hard-code.

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -16,7 +16,19 @@ jobs:
...
+      run: |
+        if  [[ "$GITHUB_REF" == "refs/heads/main" ]] ; then
+          echo We are deploying to the DEMO environment
+          app=$DEMO_CF_APP
+          title="PRIME Data Input Prototype"
+        else
+          echo We are deploying to the STAGING environment
+          app=$STAGING_CF_APP
+          title="** STAGING ** PRIME Data Input Prototype ** STAGING **"
+          echo >> $GITHUB_ENV "REACT_APP_ICON=optimus_prime.svg"
```
Comment (sharmaneil) [inline .github/workflows/deploy.yml:28]: 🚚 🤖 🇺🇸

**Review by sharmaneil — APPROVED** [review]
looks a lot cleaner to me! 

These deploy workflows have saved me SO much time circumventing cloud.gov

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -16,7 +16,19 @@ jobs:
       # VERY IMPORTANT: if you run this on any branch other than master, it will
```
Comment (emmanuel-apau) [inline .github/workflows/deploy.yml:16]: just a heads up, you may want to consider deployments to staging to trigger off git tagging, that way it avoids any accidents and resolves the need for the disclaimer, and has the added benefit of immutability

**Pull request merged**